cfillion/cfillion.cpp

```diff
@@ -367,6 +367,9 @@ int CF_EnumMediaSourceCues(PCM_source *source, const int index, double *time,
   if(name && cue.m_name)
     snprintf(name, nameSize, "%s", cue.m_name);
 
+  if (isChapter && (cue.m_flags & 4))
+      *isChapter = true;
```

Comment:
`isChapter` won't be cleared if the user calls the function with it initialized to `true`:

```
reaper.CF_EnumMediaSourceCues(src, i, 0, 0, true, '', true) -- weird usage, but it's a valid call
```
```suggestion
  if (isChapter)
      *isChapter = cue.m_flags & 4;
```

> I'm not sure if I get this comment correctly:
> > &1=DEPRECATED caller must call Extended(PCM_SOURCE_EXT_ENUMCUES, -1, &cue, 0) when finished
>
> But I think this is only relevant when setting, so probably no need to call above here?

`&1` is for the deprecated `PCM_SOURCE_EXT_ENUMCUES`. It does not apply to the newer `PCM_SOURCE_EXT_ENUMCUESEX`.

> reaper-sdk (including reaper_plugin.h) moved to GitHub recently, maybe we could add it as a submodule in future?

Yeah. They also have reaper_plugin_functions.h there. If they keep it up-to-date it could mean new SWS devs won't have to generate it themselves anymore.

The reaper_plugin.h here was slightly modified though (include paths, 437d7d0b45d8dd460062d7419d9268d671dcc3c0). Using the upstream version might require a bit of tweaking.

(The sudden move of reaper_plugin.h out of Landoleet broke ReaPack's CI builds which depended on it... ☃️ )

Reply:
Thanks, updated.
>**If** they keep it up-to-date

Yeah. :P